argilla/environment_dev.yml

```diff
@@ -67,5 +67,6 @@ dependencies:
       - ipynbname>=2023.2.0.0
       - nltk < 4.0.0
       - httpx~=0.26.0
-      # install Argilla in editable mode
+      # install Argilla packages in editable mode
+      - -e ../argilla-server[postgresql]
```

Comment:
This line has been included since the pyproject.toml does not accept configuring the argilla-server dependency as a local relative path.